source/features/latest-tag-button.tsx

```diff
@@ -70,10 +69,10 @@ const getRepoPublishState = cache.function(async (): Promise<RepoPublishState> =
 	cacheKey: () => __filebasename + ':' + getRepoURL()
 });
 
-const getAheadByCount = cache.function(async (latestTag: string): Promise<string> => {
-	const tagPage = await fetchDom(`/${getRepoURL()}/releases/tag/${latestTag}`);
+const getAheadByCount = cache.function(async (latestTag: string): Promise<string | undefined> => {
+	const tagPage = await fetchDom(`/${getRepoURL()}/releases/tag/${latestTag}`, '.release-header relative-time + a[href*="/compare/"]');
```

Comment:
Changing the selector without changing the variable name

Reply:
Sorry, I did not even think about it. I wrote this code a few days ago before you made a comment to me yesterday.
I will make sure this does not happen in the future

Reply:
Sorry, I did not even think about it. I wrote this code a few days ago before you made a comment to me yesterday.
I will make sure this does not happen in the future